1.1/reqs.ttl

```diff
@@ -0,0 +1,951 @@
+@prefix conf10ree: <http://www.opengis.net/spec/geosparql/1.0/conf/rdfs-entailment-extension/> .
+@prefix conf10tve: <http://www.opengis.net/spec/geosparql/1.0/conf/topology-vocab-extension/> .
+@prefix conf11gtx: <http://www.opengis.net/spec/geosparql/1.1/conf/geometry-topology-extension/> .
+@prefix conf11qre: <http://www.opengis.net/spec/geosparql/1.1/conf/query-rewrite-extension/> .
+@prefix conf11ree: <http://www.opengis.net/spec/geosparql/1.1/conf/rdfs-entailment-extension/> .
+@prefix conf11tve: <http://www.opengis.net/spec/geosparql/1.1/conf/topology-vocab-extension/> .
+@prefix reqs10gtx: <http://www.opengis.net/spec/geosparql/1.0/req/geometry-topology-extension/> .
+@prefix reqs10qre: <http://www.opengis.net/spec/geosparql/1.0/req/query-rewrite-extension/> .
+@prefix reqs10ree: <http://www.opengis.net/spec/geosparql/1.0/req/rdfs-entailment-extension/> .
+@prefix reqs10tve: <http://www.opengis.net/spec/geosparql/1.0/req/topology-vocab-extension/> .
+@prefix reqs11gtx: <http://www.opengis.net/spec/geosparql/1.1/req/geometry-topology-extension/> .
+@prefix reqs11qre: <http://www.opengis.net/spec/geosparql/1.1/req/query-rewrite-extension/> .
+@prefix reqs11ree: <http://www.opengis.net/spec/geosparql/1.1/req/rdfs-entailment-extension/> .
+@prefix reqs11tve: <http://www.opengis.net/spec/geosparql/1.1/req/topology-vocab-extension/> .
+@prefix conf10core: <http://www.opengis.net/spec/geosparql/1.0/conf/core/> .
+@prefix conf11core: <http://www.opengis.net/spec/geosparql/1.1/conf/core/> .
+@prefix reqs10core: <http://www.opengis.net/spec/geosparql/1.0/req/core/> .
+@prefix reqs11core: <http://www.opengis.net/spec/geosparql/1.1/req/core/> .
+
+@base <http://www.opengis.net/def/geosparql/reqs> .
+<http://www.opengis.net/def/geosparql/reqs> rdf:type owl:Ontology ;
+                                             dcterms:created "2021-06-28"^^xsd:date ;
+                                             dcterms:modified "2021-07-13"^^xsd:date ;
+                                             rdfs:seeAlso <http://www.opengis.net/doc/geosparql/1.1> ;
+                                             skos:definition "A vocabulary of the requirements and conformance classes in the GeoSPARQL 1.1 specification"@en ;
+                                             skos:prefLabel "GeoSPARQL 1.1 Requirement and Conformance Class Register" .
+
+
+
+dcterms:created rdf:type owl:AnnotationProperty .
+
+dcterms:modified rdf:type owl:AnnotationProperty .
+
+geo:testPurpose rdf:type owl:AnnotationProperty .
+
+geo:testType rdf:type owl:AnnotationProperty .
+
+skos:definition rdf:type owl:AnnotationProperty .
+
+skos:definitoin rdf:type owl:AnnotationProperty .
+
+skos:member rdf:type owl:AnnotationProperty .
+
+skos:prefLabel rdf:type owl:AnnotationProperty .
+
+prov:wasDerivedFrom rdf:type owl:AnnotationProperty .
+
+skos:Collection rdf:type owl:Class .
+
+skos:Concept rdf:type owl:Class .
+
+skos:ConceptScheme rdf:type owl:Class .

```

Comment:
I think lines 49-71 can be removed. Those terms are defined elsewhere.

Reply:
Removed